File: Hoteles-Reserva-WS/src/java/models/Reservation.java
package models;

public class Reservation {
    private String idReservation, dateCrea, dateMod;
    private Boolean active;
    private int idClient;
    private int idRoom;
    private int idPackages;

    public String getIdReservation() {
        return idReservation;
    }

    public void setIdReservation(String idReservation) {
        this.idReservation = idReservation;
    }

    public String getDateCrea() {
        return dateCrea;
    }

    public void setDateCrea(String dateCrea) {
        this.dateCrea = dateCrea;
    }

    public String getDateMod() {
        return dateMod;
    }

    public void setDateMod(String dateMod) {
        this.dateMod = dateMod;
    }

    public Boolean getActive() {
        return active;
    }

    public void setActive(Boolean active) {
        this.active = active;
    }

    public int getIdClient() {
        return idClient;
    }

    public void setIdClient(int idClient) {
        this.idClient = idClient;
    }

    public int getIdRoom() {
        return idRoom;
    }

    public void setIdRoom(int idRoom) {
        this.idRoom = idRoom;
    }

    public int getIdPackages() {
        return idPackages;
    }

    public void setIdPackages(int idPackages) {
        this.idPackages = idPackages;
    }

    
}

File: Hotel-Reservacion/Hotel-Reservacion/Models/Reservation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Hotel_Reservacion.Models
{
    public class Reservation
    {
        private String idReservation, dateCrea, dateMod;
        private bool active;
        private int idClient, idRoom, idPackages;


        public String IdReservation
        {
            get
            {
                return idReservation;
            }
            set
            {
                idReservation = value;
            }
        }
        public int IdClient
        {
            get
            {
                return idClient;
            }
            set
            {
                idClient = value;
            }
        }
        public int IdRoom
        {
            get
            {
                return idRoom;
            }
            set
            {
                idRoom = value;
            }
        }
        public int IdPackages
        {
            get
            {
                return idPackages;
            }
            set
            {
                idPackages = value;
            }
        }
        public String DateCrea
        {
            get
            {
                return dateCrea;
            }
            set
            {
                dateCrea = value;
            }
        }
        public String DateMod
        {
            get
            {
                return dateMod;
            }
            set
            {
                dateMod = value;
            }
        }
        public bool Active
        {
            get
            {
                return active;
            }
            set
            {
                active = value;
            }
        }
    }
}